### PR TITLE
Add specification from env.spec file to rpm spec file

### PR DESCRIPTION
**Pull request by lbdreyer**
@pelson

### CODE REVIEW

File: conda_rpms/build_rpm_structure.py [hunk 1]
```diff
@@ -80,12 +80,20 @@ def create_rpmbuild_for_tag(repo, tag_name, target):
     with open(manifest_fname, 'r') as fh:
         manifest = sorted(line.strip().split('\t') for line in fh)
 
+    spec_fname = os.path.join(repo.working_dir, 'env.spec')
+    if not os.path.exists(spec_fname):
+        raise ValueError("The tag '{}' doesn't have an environment specification.".format(tag_name))
+    with open(spec_fname, 'r') as fh:
```
Comment (pelson) [inline conda_rpms/build_rpm_structure.py:133]: How about just opening this with pyyaml?

Comment (bjlittle) [inline conda_rpms/build_rpm_structure.py:133]: Once https://github.com/pelson/conda-rpms/pull/5 is merged, you could just reuse https://github.com/pelson/conda-rpms/pull/5/files#diff-56c430e20547a5a55d4f6736e2e205cbR28 i.e.

``` python
spec = Config('env.spec')
...
    fh.write(generate.render_taggedenv(env_name, tag, pkgs, config, spec['env'])
```

Comment (pelson) [inline conda_rpms/build_rpm_structure.py:133]: `Config` was definitely overloaded 😄 !
I'd advise against that, and just go for `env = yaml.safe_load('env.spec').get('env', [])`.

**Comment by lbdreyer** [comment]
I'm having some rebase problems, are you able to "squash and merge" or should I keep trying to squash these?

**Pull request merged**